circuit_knitting/cutting/cut_finding/disjoint_subcircuits_state.py

```diff
@@ -430,12 +430,12 @@ def add_action(
         self,
         action_obj: DisjointSearchAction,
         gate_spec: GateSpec,
-        args: tuple | None = None,
+        *args: tuple | None,
```

Comment:
I suspect that `| None` is not actually needed.  See e.g. https://stackoverflow.com/a/37032111/1558890

Reply:
I wasn't able to look into this careful but since `mypy` didn't complain, I went ahead and made the change you suggested ([e44b4be](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/591/commits/e44b4bedb6f228347da5e602e57c24c9dee0159d)).